libs/hdf-converters/src/asff-mapper.ts

```diff
@@ -250,6 +254,58 @@ function getSecurityHub(): Record<string, Function> {
     productName
   };
 }
+// eslint-disable-next-line @typescript-eslint/ban-types
+function getTrivy(): Record<string, Function> {
+  const findingId = (finding: unknown): string => {
+    const generatorId = _.get(finding, 'GeneratorId');
+    const cveId = _.get(finding, 'Resources[0].Details.Other.CVE ID');
```

Comment:
May want to comment here why we are using Resources[0]

Reply:
explanatory comment added